app/components/course-page/header/tab-list.ts

```
@@ -103,55 +103,60 @@
+          name: 'Screencasts',
+          slug: 'screencasts',
+          internalLink: {
+            route: 'course.stage.screencasts',
+            models: this.args.currentStep.routeParams.models,
+          },
+          isActive: this.router.currentRouteName === 'course.stage.screencasts',
+        });
+      }
+
+      // @ts-ignore
+      if (this.args.currentStep.courseStage && this.args.currentStep.courseStage.course.hasConcepts) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)